src/Makefile.am

```diff
@@ -40,106 +40,106 @@ endif
-	mruset.h \
-	neuralnet.h \
-	netbase.h \
-	net.h \
-	pbkdf2.h \
-	protocol.h \
-	rpcclient.h \
-	rpcprotocol.h \
-	rpcserver.h \
-	script.h \
-	scrypt.h \
-	serialize.h \
-	strlcpy.h \
-	sync.h \
-	tally.h \
-	threadsafety.h \
-	tinyformat.h \
-	txdb.h \
-	txdb-leveldb.h \
-	ui_interface.h \
-	uint256.h \
-	util.h \
-	version.h \
-	walletdb.h \
-	wallet.h
+    addrman.h \
+    alert.h \
+    allocators.h \
+    gridcoin/appcache.h \
+    gridcoin/backup.h \
+    base58.h \
+    gridcoin/beacon.h \
+    bignum.h \
+    gridcoin/block.h \
+    gridcoin/boinc.h \
+    checkpoints.h \
+    coincontrol.h \
+    compat.h \
+    gridcoin/contract/polls.h \
+    gridcoin/contract/contract.h \
+    gridcoin/cpid.h \
+    crypter.h \
+    db.h \
+    gridcoin/fwd.h \
+    gridcoin/global_objects.hpp \
+    gridcoin/global_objects_noui.hpp \
+    gridcoin/gridcoin.h \
+    grcrestarter.h \
+    init.h \
+    kernel.h \
+    key.h \
+    keystore.h \
+    main.h \
+    miner.h \
+    mruset.h \
+    gridcoin/neuralnet.h \
+    netbase.h \
+    net.h \
+    pbkdf2.h \
+    protocol.h \
+    rpcclient.h \
+    rpcprotocol.h \
+    rpcserver.h \
+    script.h \
+    scrypt.h \
+    serialize.h \
+    strlcpy.h \
+    sync.h \
+    gridcoin/tally.h \
+    threadsafety.h \
+    tinyformat.h \
+    txdb.h \
+    txdb-leveldb.h \
+    ui_interface.h \
+    uint256.h \
+    util.h \
+    version.h \
+    walletdb.h \
+    wallet.h
 
 GRIDCOIN_CORE_CPP = addrman.cpp \
-	alert.cpp \
-	allocators.cpp \
-	appcache.cpp \
-	backup.cpp \
-	beacon.cpp \
-	block.cpp \
-	boinc.cpp \
-	checkpoints.cpp \
-	contract/polls.cpp \
-	contract/contract.cpp \
-	cpid.cpp \
-	crypter.cpp \
-	db.cpp \
-	init.cpp \
-	kernel.cpp \
-	key.cpp \
-	keystore.cpp \
-	main.cpp \
-	miner.cpp \
-	netbase.cpp \
-	net.cpp \
-	noui.cpp \
-	pbkdf2.cpp \
-	protocol.cpp \
-	contract/rpccontract.cpp \
-	rpcblockchain.cpp \
-	rpcclient.cpp \
-	rpcdump.cpp \
-	rpcmining.cpp \
-	rpcnet.cpp \
-	rpcprotocol.cpp \
-	rpcrawtransaction.cpp \
-	rpcserver.cpp \
-	rpcwallet.cpp \
-	rpcdataacq.cpp \
-	script.cpp \
-	scrypt.cpp \
-	sync.cpp \
-    tally.cpp \
-	txdb-leveldb.cpp \
-	util.cpp \
-	version.cpp \
-	wallet.cpp \
-	walletdb.cpp
+    alert.cpp \
+    allocators.cpp \
+    gridcoin/appcache.cpp \
+    gridcoin/backup.cpp \
+    gridcoin/beacon.cpp \
+    gridcoin/block.cpp \
+    gridcoin/boinc.cpp \
+    checkpoints.cpp \
+    gridcoin/contract/polls.cpp \
+    gridcoin/contract/contract.cpp \
+    gridcoin/cpid.cpp \
+    crypter.cpp \
+    db.cpp \
+    init.cpp \
+    kernel.cpp \
+    key.cpp \
+    keystore.cpp \
+    main.cpp \
+    miner.cpp \
+    netbase.cpp \
+    net.cpp \
+    noui.cpp \
+    pbkdf2.cpp \
+    protocol.cpp \
+    gridcoin/contract/rpccontract.cpp \
+    rpcblockchain.cpp \
+    rpcclient.cpp \
+    rpcdump.cpp \
+    rpcmining.cpp \
+    rpcnet.cpp \
+    rpcprotocol.cpp \
+    rpcrawtransaction.cpp \
+    rpcserver.cpp \
+    rpcwallet.cpp \
+    gridcoin/rpc/dataacq.cpp \
```

Comment:
This subfolder stuff seems too fine grained. Let's keep everything in gridcoin/ .

Reply:
@TheCharlatan Maybe. Newer bases use subfolders (rpc/, wallet/, primitives/) so it would make sense for us to mirror that structure in our on sub-tree.

Reply:
It IS overly fine-grained, but does not add that much problems. so... idont kow

Reply:
The rationale was to have the same structure regardless of if you work on the parent or our sub tree. Though we can keep our tree flat and see how it feels. If it becomes too cluttered we can split it.

Reply:
We'd anyway have to move the stuff around again, if we were to rebase to a new source, The substructure just adds another level in the directories, but not really a lot towards making it easier to grasp the code quickly.

Reply:
Alrighty. @Foggyx420 Keep the old RPC filenames and store them in gridcoin/.

Reply:
You said it nicely @TheCharlatan . I made up my mind. Flat structure in gridcoin/ subtree.